### ISSUE TITLE
Not support '#break'

### ISSUE THREAD

**Issue by SivilTaram** [purcell/airspeed#22]
Hello,thank you for your contribution,but when I wrote it as follow:
#if( $foreach.count = $people.count ) #break
But I saw that when foreach.count is equal to people.count the sequence "#break" just being printed,it wasn't executed.

**Comment by purcell** [comment]
Yeah, `#break` isn't currently supported. It'd be a bit fiddly to add, so I can't promise to implement it very quickly.
